Kireck211/adopt-me
# Fix: Improved button design

## Summary
Fixes the button design, because it was a little odd to be changed the width depending on the content of the animal card.

## Manual test
1. Pull down the changes
2. Run the command: `npm run start`
3. Verify that the button stays the same width regardless of the content of the card

## Ticket related
[Trello Ticket Fix buttons size](https://trello.com/c/gpQu06uh/13-front-fix-buttons-size-1)
